code-env/python/desc.json

```diff
@@ -4,8 +4,7 @@
     "PYTHON37",
     "PYTHON38",
     "PYTHON39",
-    "PYTHON310",
-    "PYTHON311"
+    "PYTHON310"
```

Comment:
I think we need to remove python 3.11 because it only work with torch==1.13.0 en Centos and torch==2.0.0 on MacOS which are not the torch versions supported by DSS, what do you think ?